__tests__/user/user.mutation.test.js

```diff
@@ -588,4 +588,54 @@ describe('mutations', () => {
 
     expect(res.data.deleteUser._id).toEqual(userId);
   });
+
+  test('Should change user status', async () => {
+    const result = await client.mutate({
+      mutation: gql`
+        mutation($id: ID!) {
+          switchUserStatus(id: $id) {
+            ... on SuccessfulResponse {
+              isSuccess
+            }
+            ... on Error {
+              message
+              statusCode
+            }
+          }
+        }
+      `,
+      variables: {
+        id: '9c031d62a3c4909b216e1d86',
```

Comment:
Please, don't hardcode an id, you need to create a user in `beforeAll` statement and use its id here